apache-rat-tasks/src/test/java/org/apache/rat/anttasks/ReportOptionTest.java

```diff
@@ -0,0 +1,431 @@
+            for (LicenseSetFactory.LicenseFilter filter : LicenseSetFactory.LicenseFilter.values()) {
+                    ReportConfiguration config = generateConfig(OptionCollection.LIST_LICENSES, name, filter.name());
+                    assertEquals(filter, config.listLicenses());
+            }
+        }
+
+        @Override
+        public void listFamiliesTest() {
+            String name = antName(OptionCollection.LIST_FAMILIES);
+            for (LicenseSetFactory.LicenseFilter filter : LicenseSetFactory.LicenseFilter.values()) {
+                ReportConfiguration config = generateConfig(OptionCollection.LIST_FAMILIES, name, filter.name());
+                assertEquals(filter, config.listFamilies());
+            }
+        }
+
+        @Override
+        public void noDefaultsTest() {
+            String name = antName(OptionCollection.NO_DEFAULTS);
+                ReportConfiguration config = generateConfig(OptionCollection.NO_DEFAULTS,name, "true");
+                assertTrue(config.getLicenses(LicenseSetFactory.LicenseFilter.ALL).isEmpty());
+                config = generateConfig(OptionCollection.NO_DEFAULTS,name, "false");
+                assertFalse(config.getLicenses(LicenseSetFactory.LicenseFilter.ALL).isEmpty());
+        }
+
+        @Override
+        public void outTest() {
+            File outFile = new File( baseDir, "outexample");
+            String name = antName(OptionCollection.OUT);
+            ReportConfiguration config = generateConfig(OptionCollection.OUT, name, outFile.getAbsolutePath() );
+            try (OutputStream os = config.getOutput().get()) {
+                os.write("Hello world".getBytes());
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+            try (BufferedReader reader = new BufferedReader( new InputStreamReader(Files.newInputStream(outFile.toPath())))) {
+                assertEquals("Hello world",reader.readLine());
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+        }
+
+        @Override
+        public void scanHiddenDirectoriesTest() {
+            String name = antName(OptionCollection.SCAN_HIDDEN_DIRECTORIES);
+            ReportConfiguration config = generateConfig(OptionCollection.SCAN_HIDDEN_DIRECTORIES, name, "true");
+            assertThat(config.getDirectoriesToIgnore()).isExactlyInstanceOf(FalseFileFilter.class);
+        }
+
+        @Override
+        public void styleSheetTest() {
+            String name = antName(OptionCollection.STYLESHEET_CLI);
+            URL url = ReportTest.class.getResource("MatcherContainerResource.txt");
+            if (url == null) {
+                fail("Could not locate 'MatcherContainerResource.txt'");
+            }
+            for (String sheet : new String[]{"target/optionTools/stylesheet.xlt", "plain-rat", "missing-headers", "unapproved-licenses", url.getFile()}) {
```

Comment:
is this a typo or does the file just have in invalid name? stylesheet.xstl instead of xlt?